collect_app/src/main/java/org/odk/collect/android/preferences/qr/QRScannerFragment.java

```diff
@@ -0,0 +1,138 @@
+/* Copyright (C) 2017 Shobhit
```

Comment:
Did this come from a file that had a copyright notice on it? 

@lognaturel should we have this kind of copyright notice in the code? I know copyright headers like this can be useful for relicensing projects (as you can scan for all the copyright holders) but as far as I know we don't have them for every contribution so we'd probably need to get that info from Git anyway.

Reply:
https://github.com/opendatakit/governance/issues/2 For unresolved governance issue. I go back and forth on what I think is best and so have stopped making any code review comments. I think that yes, git is and should be the ultimate source for authorship and copyright info. I also think that for forks, it's nice to have an ongoing reminder that they're pulling in code from another source. I find the license header more compelling than the copyright notice. I (intend to) always include the header on new files I create.

This specific file was renamed. On one hand, the header does capture original copyright. On the other, the file has changed enough that it's not super meaningful.

So I don't have strong feelings about what happens to this header.

Reply:
Yeah totally agree a license header is a different story. I'd be up for getting rid of author specific copyright notices like this where we see them as they're usually just inaccurate - they don't have copyright for this entire file. I'd generally ask people not to include them unless they feel strongly they want it as part of the PR.

Getting rid of them without lawyers is pretty scary but I'd say we can remove this as it looks autogenerated and doesn't actually reference anyone by email/full name. I'll CC our pal @shobhitagarwal1612 (who I'm guessing is our "shobit" here) and see if he's ok getting rid of it. At least that will be one gone!

Reply:
I have no issues. Please remove it. Happy to see the progress happening to the project 😄 🚀 

I have always been nervous while adding a new file without a copyright. But since that issue is still a gray area, I'd leave the final decision to @lognaturel.